design-documents/graph-ql/coverage/customer-email-password-update.md

```diff
@@ -36,15 +36,15 @@ Deprecate `updateCustomer` mutation in favor of `updateCustomerV2`. `CustomerUpd
  
  ```graphql
   mutation {
-     updateCustomerEmail(email: String!, password: String!): Customer
+     updateCustomerEmail(email: String!, password: String!): CustomerOutput
   }
  ```
  
  **updateCustomerPassword**
  
  ```graphql
   mutation {
-     updateCustomerPassword(password: String!, old_password: String!): Customer
+     updateCustomerPassword(currentPassword: String!, newPassword: String!): CustomerOutput
```

Comment:
According to https://github.com/magento/architecture/issues/269, fields should be snake case. I would also propose to keep password and old_password arguments.

Reply:
But this is argument of the mutation.
Regarding naming, I think we need more thoughts here.
@buskamuza @paliarush